src/components/AssetInput/AssetThumbnail.tsx

```diff
@@ -0,0 +1,47 @@
+import IconAudio from "@foundation/Icon/Generated/IconAudio";
+import { IconSize } from "@foundation/Icon/IconSize";
+import { merge } from "@utilities/merge";
+import React, { cloneElement, FC } from "react";
+import { AssetInputProps, AssetInputSize } from "./AssetInput";
+import { SelectedAssetProps } from "./SingleAsset/SelectedAsset";
+
+type AssetThumbnailProps = {
+    asset: SelectedAssetProps["asset"];
+    size: AssetInputProps["size"];
+    isActive: boolean;
+};
+type isMultiAsset = {
+    isMultiAsset?: boolean;
+};
+export const AssetThumbnail: FC<Required<AssetThumbnailProps> & isMultiAsset> = ({
```

Comment:
I find the typing a bit confusing, so I'd suggest the following instead: 
```suggestion
type AssetThumbnailProps = {
    asset: AssetType;
    size: AssetInputSize;
    isActive?: boolean;
    isMultiAsset?: boolean;
};

export const AssetThumbnail: FC<AssetThumbnailProps> = ({
```

Reply:
done